setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="rhumba",
-    version='0.1.18',
+    version='0.2.0',
```

Comment:
Non-release revisions probably shouldn't use "production" version numbers. I prefer to have `1.2.3a0` between releases `1.2.2` and `1.2.3` (and update it to `1.3.0a0` if we make changes big enough to move to `1.3.0` on the next release). That also has the nice benefit that the tagged commit for the release is just the version number change.